### PR TITLE
language: add entity sentiment sample

### PR DESCRIPTION
**Pull request by broady**
For this page:
https://cloud.google.com/natural-language/docs/beta

### CODE REVIEW

File: language/analyze/analyze_test.go [hunk 1]
```diff
@@ -28,6 +30,25 @@ func TestSentiment(t *testing.T) {
...
+	testutil.SystemTest(t)
+	ctx, c := newBetaClient(t)
+
+	res, err := analyzeEntitySentiment(ctx, c, "Marvin Gaye is the best")
+	if err != nil {
+		t.Fatalf("got %v, want nil err", err)
+	}
+	for _, e := range res.Entities {
+		if e.Name == "Marvin Gaye" {
+			if got := e.Sentiment.Score; got <= 0 {
```
Comment (rakyll) [inline language/analyze/analyze_test.go]: What's a negative score?

Comment (broady) [reply language/analyze/analyze_test.go]: sentiment. like "Marvin Gaye has no talent"

File: language/analyze/README.md [hunk 2]
```diff
@@ -17,15 +17,15 @@
 ## Run the sample
 
 ```bash
-go run analyze.go <command> <text>
+go build -o analyze && ./analyze <command> <text>
```
Comment (rakyll) [inline language/analyze/README.md]: Can we just omit the -o flag? The binary name will be analyze anyways.

Comment (broady) [reply language/analyze/README.md]: yeah, i thought to make it a bit more explicit. done

**Pull request merged**